### DISCUSSION TITLE
Você pode criar uma extensão para o Google chorme que faça a leitura da transcrição usando o Read a load?

### DISCUSSION THREAD

**Discussion by lucasdasilvaborges** [ken107/read-aloud#270] [Ideas]
https://voiceovermaker.io/pt/transcri%C3%A7%C3%A3o-e-tradu%C3%A7%C3%A3o-de-v%C3%ADdeo-e-%C3%A1udio

**Comment by ken107** [comment]
This would make for a good Chrome extension.  Not as a feature of Read Aloud, but as a separate Chrome extension.  Maybe you can call it "Read Aloud YouTube Transcript".

But it will somewhat tricky to implement.  You'll need to monitor the YouTube playback progress, figure out which line needs to be spoken aloud using the timestamps, figure out how fast it should speak based on how many seconds until the next line is spoken.
